app/src/main/java/com/rockthevote/grommet/data/db/dao/SessionDao.kt

```diff
@@ -20,4 +18,7 @@ interface SessionDao {
     @Transaction
     @Query("SELECT * FROM session")
     fun getSessionWithRegistrations(): List<SessionWithRegistrations>
+
+    @Update(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
I tried this for my insert method and got a unique constraint violation. Wasn't really sure how given that i deleted everything prior to running it.

Does an update method even require a conflict strategy?

Reply:
Hmm, that's a good point for `Update`, I can't see a case where `onConflict` is needed is needed. I'm not sure why you would have got the constraint violation, was the Primary Key pre-populated?

Reply:
no, don't think so